File: frontend/web-delta/src/app/models/patrimony/patrimony.model.ts
import {BaseModel} from '../base/base.model';

export class PatrimonyModel extends BaseModel {

  private _value: Number;
  private _description: String;
  private _customer_dossier: Number;

    /**
     * Getter value
     * @return {Number}
     */
  public get value(): Number {
    return this._value;
  }


    /**
     * Getter customer
     * @return {Number}
     */
  public get customer_dossier(): Number {
    return this._customer_dossier;
  }

    /**
     * Setter value
     * @param {Number} value
     */
  public set value(value: Number) {
    this._value = value;
  }

    /**
     * Setter customer
     * @param {Number} value
     */
  public set customer_dossier(value: Number) {
    this._customer_dossier = value;
  }

    /**
     * Getter description
     * @return {String}
     */
	public get description(): String {
		return this._description;
	}

    /**
     * Setter description
     * @param {String} value
     */
	public set description(value: String) {
		this._description = value;
	}

}

File: backend/server_delta/server_delta_app/models/patrimony/patrimony_model.py
from django.db import models
from server_delta_app import models as server_delta_app_models, managers


class PatrimonyModel(server_delta_app_models.BaseModel):
    """
        Model that represent the customer debt registered in app
    """
    class Meta:
        db_table = "patrimony"

    value = models.DecimalField(max_digits=10, decimal_places=2)
    description = models.CharField(max_length=1000)
    customer_dossier = models.ForeignKey(server_delta_app_models.CustomerDossierModel, on_delete=models.CASCADE, related_name='patrimonies')

    objects = managers.PatrimonyManager()
